components/layout_2020/display_list/stacking_context.rs

```diff
@@ -476,6 +499,123 @@ impl BoxFragment {
             );
         }
     }
+
+    fn build_reference_frame_if_necessary(
+        &self,
+        builder: &mut DisplayListBuilder,
+        border_rect: &PhysicalRect<Length>,
+    ) -> bool {
```

Comment:
Please add a doc-comment to explain what this return value means.

Reply:
Okay.